releases/v1.4.0-rc.toml

```diff
@@ -47,9 +47,10 @@ while driving toward ever increasing stability.
 * **Split UID and GID namespace mapping in spec generation** [#3881](https://github.com/containerd/containerd/pull/3881)
 * **Propagate snapshotter layer annotations on unpack** [#3911](https://github.com/containerd/containerd/pull/3911)
 * **Create image record after blob download to fix concurrent download issue** [#3972](https://github.com/containerd/containerd/pull/3972)
+* **Use spec's mountLabel when mounting container rootfs** [#4051](https://github.com/containerd/containerd/pull/4051)
 * **Add Linux resources to spec options** [#4083](https://github.com/containerd/containerd/pull/4083)
 * **New registry configuration tooling for improved mirror and endpoint options** [#4138](https://github.com/containerd/containerd/pull/4138)
-* **Use spec's mountLabel when mounting container rootfs** [#4051](https://github.com/containerd/containerd/pull/4051)
+* **Add namespace query parameter when using a registry proxy** [#4413](https://github.com/containerd/containerd/pull/4413)
 
```

Comment:
* Deperecate io.containerd.runtime.v1.* and io.containerd.runc.v1 https://github.com/containerd/containerd/pull/4384
* libseccomp is no longer required. The seccomp support is always built-in, and
 "seccomp" build tag is now no-op. https://github.com/containerd/containerd/pull/4439